source/core_http_client.c

```diff
@@ -195,13 +192,34 @@ static HTTPStatus_t getFinalResponseStatus( HTTPParsingState_t parsingState,
  * @param[in] pResponse Response message to receive data from the network.
  * @param[in] pRequestHeaders Request headers for the corresponding HTTP request.
  *
- * @return Returns #HTTPSuccess if successful. Please see #receiveHttpData,
- * #parseHttpResponse, and #getFinalResponseStatus for other statuses returned.
+ * @return Returns #HTTPSuccess if successful. #HTTPNetworkError for a transport
+ * receive error. Please see #parseHttpResponse, and #getFinalResponseStatus for
```

Comment:
Don't think Oxford comma is needed here
```suggestion
 * receive error. Please see #parseHttpResponse and #getFinalResponseStatus for
```

Reply:
two item lists don't require commas

Reply:
Sorry left it after delting a third item